svm/src/transaction_processor.rs

```diff
@@ -309,62 +297,128 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
             program_cache_for_tx_batch
         });
 
-        let (loaded_transactions, load_accounts_us) = measure_us!(load_accounts(
+        let account_keys_in_batch = sanitized_txs
+            .iter()
+            .flat_map(|tx| tx.account_keys().iter())
+            .sorted()
+            .dedup()
+            .count();
```

Comment:
we could just over-allocate here using account keys lengths, instead of doing an intermediate allocation for the sorting and deduping. 

that will definitely give us enough room, and even in our worst expected case (for validator) of 64 txs using 64 accts it's only 4096 pubkeys.